ms2query/benchmarking/create_figures_benchmarking.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Tuple, Dict
```

Comment:
Same as other benchmarking script. Briefly mention how this is used (to create figures for the paper).